### ISSUE TITLE
Get without callback returns incorrect signature

### ISSUE THREAD

**Issue by simonbrent** [memcachier/memjs#141]
The documentation for `Client.prototype.get` states:
> // The callback signature is:
//
//     callback(err, value, flags)

but the code contains the following:
```
  if(callback === undefined) {
    return promisify(function(callback) {
      self.get(key, function(err, value, flags) {
        callback(err, {value: value, flags: flags});
      });
    });
  }
```
Changing this to 
```
  if(callback === undefined) {
    return promisify(function(callback) {
      self.get(key, callback)
    });
  }
 ```
would fix this issue.

**Comment by alevy** [comment]
Indeed! Can you send a pull request with this change? The alternative would be to change the comment, but the documented behavior seems like a better interface.

**Comment by simonbrent** [comment]
I have tried, but it is more problematic than I thought, since you can't return multiple values from a Promise. I therefore don't think there's a way to be consistent with the signatures between the promisified version and the unpromisified version without changing the unpromisified version, which I imagine would be undesirable.
